interfaces.go

```diff
@@ -69,7 +76,9 @@ type ArchiverAsync interface {
 	// Use ArchiveAsync if you can't pre-assemble a list of all
 	// the files for the archive. Close the files channel after
 	// all the files have been sent.
-	ArchiveAsync(ctx context.Context, output io.Writer, files <-chan File) error
+	//
+	// This won't return until the channel is closed.
+	ArchiveAsync(ctx context.Context, output io.Writer, files <-chan ArchiveAsyncJob) error
```

Comment:
```suggestion
	ArchiveAsync(ctx context.Context, output io.Writer, jobs <-chan ArchiveAsyncJob) error
```

Wondering if maybe we should just keep this consistent with implementations.

Reply:
Ah yes, well spotted. Have fixed that now :-)